README.md

```diff
@@ -122,6 +122,7 @@ Contributors:
 - [Thomas Stieglmaier](https://stieglmaier.me/)
 - [Martin Yankov](https://github.com/marto97)
 - [Ilja Zakharov](https://github.com/IljaZakharov)
+- [Levente Bajczi](https://github.com/leventeBajczi)
```

Comment:
You definitively should be listed here! I would just ask you to insert the name in the alphabetical ordering.

Reply:
Oh, I did not realize it was ordered! Will fix shortly.